test/unittests/mongoBackend/mongoRegisterContext_update_test.cpp

```diff
@@ -76,11 +76,8 @@ extern void setMongoConnectionForUnitTest(DBClientBase*);
 * registrations collection.
 *
 */
-static void prepareDatabase(void) {
-
-  /* Set database */
-  setupDatabase();
```

Comment:
The `setupDatabase()` call in the original code has been moved or removed? It seems the second case, in which case this goes beyond just style check... or maybe I'm missing something.